public/js/main.js

```diff
@@ -39,7 +39,8 @@
     }
 
     kitten_imagery.setAttribute('class', 'kitten')
-
+    kitten_imagery.setAttribute('href', window.kittens[index].url)
```

Comment:
Ok, I see. Miscommunication.  Only the Giphy attribution on the About page is necessary.